en/flight_controller/kakutef7.md

```diff
@@ -94,6 +94,18 @@ Parameter | Setting
 [MOT_ORDERING](../advanced_config/parameter_reference.md#MOT_ORDERING) | If you use a 4-in-1 ESC with Betaflight/Cleanflight motor assignment, this parameter can be set accordingly.
 
 
+## Serial Port Mapping
+
+UART | Device | Port
+--- | --- | ---
+USART1 | /dev/ttyS0 | TELEM1
+USART2 | /dev/ttyS1 | TELEM2
+USART3 | /dev/ttyS2 | Debug Console
+UART4 | /dev/ttyS3 | GPS1
+USART6 | /dev/ttyS4 | RC SBUS
+UART7 | /dev/ttyS5 | ESC telemetry
```

Comment:
This last one is from  hamishwillee-serial_port_mapping. Dunno what ESC telemetry is!

Reply:
This is @bkueng baby, but I think it is the Serial feedback from the ESC when used in D-shot.

Reply:
Seems likely. I've added that in brackets afterwards. Marking this as resolved. @bkueng if this is something different let me know.

Reply:
Yes exactly. It goes directly to the ESC.